### ISSUE TITLE
Neurogenic bladder

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#11428]
**Preferred term label:**
Neurogenic bladder

-- organize with two subterms, spastic and flaccid NB
-- add definition and move existing term https://hpo.jax.org/browse/term/HP:0005340

**Comment by pnrobinson** [comment]
added
